.github/workflows/push.yml

```diff
@@ -0,0 +1,62 @@
+name: Build and push Docker images
+
+on:
+  push:
+    branches: 'main'
+
+jobs:
+  ubuntu-arm64:
+    strategy:
+      matrix:
+        distro:
+          - focal
+          - jammy
+          - latest
+    runs-on: oqs-arm64
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Login to Docker Hub
```

Comment:
Is this needed on a non-pushing action?

Reply:
This is a pushing action that will only run on `main`.

Reply:
Sorry for the noise: I was convinced I saw and commented on this in the "non-pushing" build job: In the final push it obviously is needed.